fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraVersioningIT.java

```diff
@@ -153,6 +154,61 @@ public void testCreateVersion() throws Exception {
         }
     }
 
+    @Test
+    public void testCreateVersionWithSlugHeader() throws Exception {
+        createVersionedContainer(id, subjectUri);
+
+        // Bad request with Slug header to create memento
+        final String mementoDateTime = "Tue, 3 Jun 2008 11:05:30 GMT";
+        final String body = "<" + subjectUri + "> <info:test#label> \"bar\"";
+        final HttpPost post = postObjMethod(id + "/" + FCR_VERSIONS);
+
+        post.addHeader("Slug", "version_label");
+        post.addHeader(MEMENTO_DATETIME_HEADER, mementoDateTime);
+        post.addHeader(CONTENT_TYPE, "text/n3");
+        post.setEntity(new StringEntity(body));
+
+        assertEquals("Created memento with Slug!",
+                BAD_REQUEST.getStatusCode(), getStatus(post));
+    }
+
+    @Test
+    public void testCreateVersionWithMementoDatetimeFromat() throws Exception {
+        createVersionedContainer(id, subjectUri);
+
+        // Create memento with RFC-1123 date-time format
+        final String mementoDateTime = "Tue, 3 Jun 2008 11:05:30 GMT";
+        final String body = "<" + subjectUri + "> <info:test#label> \"bar\"";
```

Comment:
Once #1300 is merged this test will need to submit an entire record instead of a single triple, but whether you will need to update it depends on the order things get merged.

Reply:
@bbpennel I'll wait until your PR(s) are merged to fix it then. Thanks.